### ISSUE TITLE
Bias term in conv layers of Resnet

### ISSUE THREAD

**Issue by NicolasHug** [pytorch/vision#4914]
The [Basic block class](https://github.com/pytorch/vision/blob/348f75ceb5b971dda7a2695c285bd5f8d4277069/torchvision/models/resnet.py#L57) of the ResNet model creates convolution layers followed by a normalization layer, which is specified by the user with the `norm_layer` parameter.

The bias term of these convolutions layers is hardcoded to False:

https://github.com/pytorch/vision/blob/348f75ceb5b971dda7a2695c285bd5f8d4277069/torchvision/models/resnet.py#L38-L49

This is expected if `norm_layer` is a BatchNorm layer, but I'm wondering if this is correct if the user passes e.g. an `InstanceNorm2d` layer, for which the `affine` parameter is False by default? I'm reading different opinions on this

Some related discussions here: https://datascience.stackexchange.com/questions/46407/conv-bias-or-not-with-instance-normalization and here https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/issues/981

Maybe a fix would be to pass `affine=True` when instanciating the `norm_layer`, but I don't know if all relevant norm layers accept this parameter.

**Comment by NicolasHug** [comment]
Discussing this offline with @datumbox : the bias term in the conv layers is made redundant by the **normalization** of the norm layer. Whether the norm layer has its own bias term (affine parameter) is a different thing.
All norm layers normalize the input in such a way that the bias term gets canceled out, so there's no problem here.
Thanks for the discussion @datumbox
